filter/filter.go

```diff
@@ -165,9 +164,18 @@ func (c *Client) UpdateFilterOutputBytes(ctx context.Context, userAuthToken, ser
 		return err
 	}
 
-	headers.SetAuthToken(req, userAuthToken)
-	headers.SetServiceAuthToken(req, serviceAuthToken)
-	headers.SetDownloadServiceToken(req, downloadServiceToken)
+	err = headers.SetAuthToken(req, userAuthToken)
+	if err != nil {
+		return err
+	}
+	err = headers.SetServiceAuthToken(req, serviceAuthToken)
```

Comment:
Can use `if err = .. ` shorthand here on these lines and below

Reply:
Also could you please wrap each of these errors (and basically all errors you return ever!) with fmt.Errorf and brief descriptive context? i.e.:
`return fmt.Errorf("failed to set download service token header: %w", err)`

Reply:
FYI i was just fixing the unhandled errors as a first pass to just make the code more robust ...